integrationtest/performance/backend/elasticsearch/src/main/java/org/hibernate/search/integrationtest/performance/backend/elasticsearch/testsupport/ElasticsearchBackendHolder.java

```diff
@@ -44,6 +45,9 @@ protected ConfigurationPropertySource getDefaultBackendProperties(TemporaryFileH
 
 		map.put( ElasticsearchIndexSettings.SCHEMA_MANAGEMENT_MINIMAL_REQUIRED_STATUS, IndexStatus.YELLOW );
 
+		// In some machine (e.g. MacOS machines) there is an extra latency
+		map.put( ElasticsearchBackendSettings.READ_TIMEOUT, "120000" );
```

Comment:
Let's stop a minute and ask ourselves why a single request would need a 2 minute timeout, shall we? Let's investigate the test that prompted you to add this, on Zulip.